playbooks/common/openshift-master/config.yml

```diff
@@ -212,6 +212,11 @@
       tasks_from: master
     when: openshift_use_kuryr | default(false) | bool
 
+  - include_role:
```

Comment:
We've already had one issue filed with users being confused about the bootstrap code being put in-line with normal provisioning.

We should add a comment to anything that references bootstrapping to clarify to users that this is not meant to be used during the normal install.